pkgs/development/python-modules/mayim/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, wheel
+
+, psycopg
+, aiosqlite
+, asyncmy
+
+# test
+, coverage
+, pytest
+, pytest-cov
+, pytest-asyncio
+, tox
+
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "mayim";
+  version = "1.1.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "ahopkins";
+    repo = pname;
```

Comment:
nits: Since `repo` and `pname` are semantically different, `repo = "mayim"` is preferred rather than `repo = pname`.